File: 0171-Excel-Sheet-Column-Number/0171.py
class Solution:
    def titleToNumber(self, columnTitle: str) -> int:
        ans = 0

        for i in range(len(columnTitle)):
            # convert charater to integer: A -> 1 (65 - 64)
            ans += (ord(columnTitle[i]) - 64) * (26 ** (len(columnTitle) - i - 1))

        return ans 

File: 0171-Excel-Sheet-Column-Number/0171.java
class Solution {
    public int titleToNumber(String columnTitle) {
        int ans = 0;
        int len = columnTitle.length();

        for (int i = 0; i < len; i++) {
            ans += ((int) columnTitle.charAt(i) - 64) * Math.pow(26, len - i - 1);
        }

        return ans;
    }
}